### ISSUE TITLE
use "title" element as name for display

### ISSUE THREAD

**Issue by paregorios** [DCLP/dclpxsltbox#206]
[In February 2016](https://github.com/DCLP/dclpxsltbox/issues/173), it was envisioned that the contents of the ```<title>``` element in the TEI header would be used as the "display name", presumably in the HTML output. Presently, the ```<h3>``` element on the HTML page for a DCLP text uses

For example, [in the xml for TM59749](https://github.com/DCLP/idp.data/blob/dclp/DCLP/60/59749.xml#L7), we find:

```xml
<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="m59749">
  <teiHeader>
      <fileDesc>
         <titleStmt>
            <title>P.Herc. 419, 697, 1634</title>
```

but [in the HTML](http://litpap.info/dclp/59749) we see:

```html
<div id="main">
   <div class="content ui-corner-all">
      <h3 style="text-align:center">Trismegistos <a href="http://www.trismegistos.org/text/59749">59749</a>
            = <a href="http://www.trismegistos.org/ldab/text.php?quick=853">LDAB 
            853</a></h3>
```

which renders thus:

![screen shot 2017-02-10 at 4 03 11 pm](https://cloud.githubusercontent.com/assets/263285/22845899/7a0e069e-efaa-11e6-9fed-677509ab81ce.png)

The XSLT that produces the content of the ```<h3>``` element is found [in the named template "get-references" in MakeHTML.xsl](https://github.com/DCLP/navigator/blob/dclp/pn-xslt/MakeHTML.xsl#L489). It would be simple to replace this behavior with something that grabbed the contents of the ```<title>``` element in the header, as specified above; however, I wonder if anyone will miss having the parallel references available.

So this is a question for @rla2118, @HolgerEssler, and @jcowey: should we proceed with replacing the current behavior, or should we come up with a hybrid solution?

A second question: what is the priority on this task, and does it belong in the WB milestone?

**Comment by HolgerEssler** [comment]
I suggest a hybrid solution, similar to papyri.info where we would have e.g.
p.wuerzb.5 = HGV P.Würzb. 5 = Trismegistos 5533

Thus on litpap.info we should have something like

<h3 style="text-align:center">P.Herc. 419, 697, 1634 = Trismegistos <a 
href="http://www.trismegistos.org/text/59749">59749</a>
             = <a 
href="http://www.trismegistos.org/ldab/text.php?quick=853">LDAB
             853</a></h3>

This would mean to concat the title string with the current content of 
the h3 element.

**Comment by paregorios** [comment]
@HolgerEssler thanks, that will work well and will be easy to implement.  Do you think this is a priority for the Wb Workshop?

**Comment by HolgerEssler** [comment]
For me it is not a priority for WB, but I am happy to leave the decision 
to James and Rodney.

**Comment by paregorios** [comment]
nudge @rla2118 @jcowey ^^^

**Comment by rla2118** [comment]
This is good.  I agree with Holger, though, that it's not a priority.

**Comment by paregorios** [comment]
ok let's leave it on the backlog for now

**Comment by paregorios** [comment]
Somehow this is working on navigator master branch. But somehow (as reported by @rla2118), #225 breaks that functionality despite the fact that it was cloned from master. I'm taking this to investigate.

**Comment by paregorios** [comment]
blocker for #225

**Comment by paregorios** [comment]
This is not to be confused with the rendering of ```<titleStmt><title>``` under the "Content" rubric, which currently works.

**Comment by paregorios** [comment]
Moved this comment from @rla2118 here from #225: 

> We do not plan to populate "Contents" going forward. Right, @jcowey?

**Comment by paregorios** [comment]
assigning to @jcowey for response to @rla2118, which will clarify what we need to do with the XSLT

**Comment by rla2118** [comment]
This needs some discussion, since I'm confused by the term "display name," which is a rubric in the editor. Can we look at it next Wednesday? @HolgerEssler @jcowey @paregorios

**Comment by paregorios** [comment]
@jcowey could this issue be added please to the wiki agenda for next week's meeting?

**Comment by jcowey** [comment]
Done.
http://147.142.225.252/paptrac/wiki/dclpRegSkypeMeetings

**Comment by paregorios** [comment]
Thanks!

-- 
Tom Elliott, Ph.D.
Associate Director for Digital Programs and Senior Research Scholar
Institute for the Study of the Ancient World (NYU)
http://isaw.nyu.edu/people/staff/tom-elliott

**Comment by paregorios** [comment]
A different solution, pulling in the principal edition, P.Herc reference, etc. was proposed by @jcowey and has been implemented with #269 and #212. Closing as superseded by those tickets.
